src/prefect/tasks/dropbox/dropbox.py

```diff
@@ -0,0 +1,49 @@
+import dropbox
+
+from prefect.client import Secret
+from prefect.core import Task
+from prefect.utilities.tasks import defaults_from_attrs
+
+
+class DropboxDownload(Task):
+    """
+    Task for downloading a file from Dropbox. Note that _all_ initialization settings can be
+    provided / overwritten at runtime.
+
+    Args:
+        - path (str): the path to the file to download
+        - access_token_secret (str, optional): the name of the Prefect Secret containing a
+            Dropbox access token; defaults to `"DROPBOX_ACCESS_TOKEN"`
+        - **kwargs (optional): additional kwargs to pass to the `Task` constructor
+    """
+
+    def __init__(self, path: str, access_token_secret: str = None, **kwargs):
+        self.path = path
```

Comment:
It's interesting to me that you _require_ `path` but _also_ allow it to be overwritten at runtime?  What's the reasoning for that pattern?

Reply:
This is actually an oversight!